lib/fishjam_web/api_spec/component/hls.ex

```diff
@@ -100,12 +70,7 @@ defmodule FishjamWeb.ApiSpec.Component.HLS do
           description: "Whether the video is stored after end of stream",
           default: false
         },
-        s3: %Schema{
-          type: :object,
-          description: "Credentials to AWS S3 bucket.",
-          oneOf: [S3],
-          nullable: true
-        },
+        s3: S3.schema(),
```

Comment:
This isn't the same as before -- the field is no longer nullable. Why this change?